apps/mux/.gitignore

```diff
@@ -0,0 +1 @@
+.now
```

Comment:
As we use our deployment pipeline and hosting it would be nice to drop all mentions of now.